src/routes/ar-io.ts

```diff
@@ -243,14 +243,14 @@ arIoRouter.post(
         return;
       }
 
-      const parquetExporter = await ParquetExporter.create({
+      const parquetExporter = new ParquetExporter({
         log,
         duckDbPath: 'data/duckdb/db.duckdb',
-        sqliteBundlesDbPath: 'data/sqlite/bundles.db',
-        sqliteCoreDbPath: 'data/sqlite/core.db',
+        bundlesDbPath: 'data/sqlite/bundles.db',
+        coreDbPath: 'data/sqlite/core.db',
       });
 
-      await parquetExporter.export({
+      parquetExporter.export({
```

Comment:
@djwhitt now we're not waiting for the export to finish to respond as it can take hours

Reply:
Yeah, that's fine. As you mentioned in DMs, having an endpoint to check status would be good, but we can handle that in a follow up ticket.